dlut-cite-style.csl

```diff
@@ -0,0 +1,230 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" name-delimiter=", " delimiter-precedes-last="always" initialize-with=" " name-as-sort-order="all" sort-separator=" " names-delimiter=". " demote-non-dropping-particle="never" default-locale="zh-CN">
+  <!-- This style was edited with the Visual CSL Editor (https://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>DLUT cite style (numeric, Multi-lang)</title>
+    <title-short>DLUT</title-short>
+    <id>http://www.zotero.org/styles/dlut-cite-style</id>
+    <link href="http://www.zotero.org/styles/dlut-cite-style" rel="self"/>
+    <link href="http://www.std.gov.cn/gb/search/gbDetailed?id=71F772D8055ED3A7E05397BE0A0AB82A" rel="documentation"/>
```

Comment:
Do you know which style you used as a base. You should include a link with with `rel="template"` to acknowledge the work someone else did already.

Reply:
The documentation link does not work